2016/09/08 17:16:55 [emerg] 21199#0: "access_log" directive is not allowed here in /www/doc/Lua/openresty-test/conf/nginx.conf:6
2016/09/08 17:19:13 [emerg] 21292#0: bind() to 0.0.0.0:8181 failed (98: Address already in use)
2016/09/08 17:19:13 [emerg] 21292#0: bind() to 0.0.0.0:8181 failed (98: Address already in use)
2016/09/08 17:19:13 [emerg] 21292#0: bind() to 0.0.0.0:8181 failed (98: Address already in use)
2016/09/08 17:19:13 [emerg] 21292#0: bind() to 0.0.0.0:8181 failed (98: Address already in use)
2016/09/08 17:19:13 [emerg] 21292#0: bind() to 0.0.0.0:8181 failed (98: Address already in use)
2016/09/08 17:19:13 [emerg] 21292#0: still could not bind()
2016/09/10 23:06:06 [error] 41239#0: *6097 failed to load external Lua file "/Users/bug1024/Code/doc/Lua/openresty-test/hi.lua": cannot open /Users/bug1024/Code/doc/Lua/openresty-test/hi.lua: No such file or directory, client: 127.0.0.1, server: , request: "GET /app/hi HTTP/1.1", host: "127.0.0.1:8181"
2016/09/10 23:06:15 [error] 41239#0: *6098 failed to load external Lua file "/Users/bug1024/Code/doc/Lua/openresty-test/hi.lua": cannot open /Users/bug1024/Code/doc/Lua/openresty-test/hi.lua: No such file or directory, client: 127.0.0.1, server: , request: "GET /app/hi/ HTTP/1.1", host: "127.0.0.1:8181"
2016/09/10 23:06:46 [error] 41239#0: *6099 failed to load external Lua file "/Users/bug1024/Code/doc/Lua/openresty-test/hello.lua": cannot open /Users/bug1024/Code/doc/Lua/openresty-test/hello.lua: No such file or directory, client: 127.0.0.1, server: , request: "GET /app/hello HTTP/1.1", host: "127.0.0.1:8181"
2016/09/10 23:06:53 [error] 41239#0: *6099 failed to load external Lua file "/Users/bug1024/Code/doc/Lua/openresty-test/hello.lua": cannot open /Users/bug1024/Code/doc/Lua/openresty-test/hello.lua: No such file or directory, client: 127.0.0.1, server: , request: "GET /app/hello/ HTTP/1.1", host: "127.0.0.1:8181"
2016/09/10 23:07:46 [error] 41281#0: *6104 failed to load external Lua file "/Users/bug1024/Code/doc/Lua/openresty-test/hi.lua": cannot open /Users/bug1024/Code/doc/Lua/openresty-test/hi.lua: No such file or directory, client: 127.0.0.1, server: , request: "GET /app/hi/ HTTP/1.1", host: "127.0.0.1:8181"
2016/09/10 23:11:23 [error] 41332#0: *1 failed to load external Lua file "/Users/bug1024/Code/doc/Lua/openresty-test/hello.lua": cannot open /Users/bug1024/Code/doc/Lua/openresty-test/hello.lua: No such file or directory, client: 127.0.0.1, server: , request: "GET /app/hello/ HTTP/1.1", host: "127.0.0.1:8181"
2016/09/10 23:11:29 [error] 41332#0: *1 failed to load external Lua file "/Users/bug1024/Code/doc/Lua/openresty-test/hello.lua": cannot open /Users/bug1024/Code/doc/Lua/openresty-test/hello.lua: No such file or directory, client: 127.0.0.1, server: , request: "GET /app/hello HTTP/1.1", host: "127.0.0.1:8181"
2016/09/10 23:12:45 [error] 41356#0: *4 failed to load external Lua file "/Users/bug1024/Code/doc/Lua/openresty-test/hello.lua": cannot open /Users/bug1024/Code/doc/Lua/openresty-test/hello.lua: No such file or directory, client: 127.0.0.1, server: , request: "GET /hello/ HTTP/1.1", host: "127.0.0.1:8181"
2016/09/10 23:24:22 [emerg] 41464#0: bind() to 0.0.0.0:8181 failed (48: Address already in use)
2016/09/10 23:24:22 [emerg] 41464#0: bind() to 0.0.0.0:8181 failed (48: Address already in use)
2016/09/10 23:24:22 [emerg] 41464#0: bind() to 0.0.0.0:8181 failed (48: Address already in use)
2016/09/10 23:24:22 [emerg] 41464#0: bind() to 0.0.0.0:8181 failed (48: Address already in use)
2016/09/10 23:24:22 [emerg] 41464#0: bind() to 0.0.0.0:8181 failed (48: Address already in use)
2016/09/10 23:24:22 [emerg] 41464#0: still could not bind()
2016/09/10 23:35:09 [error] 41773#0: *11 failed to load external Lua file "/Users/bug1024/Code/doc/Lua/openresty-test/app/hel.lua": cannot open /Users/bug1024/Code/doc/Lua/openresty-test/app/hel.lua: No such file or directory, client: 127.0.0.1, server: , request: "GET /app/hel HTTP/1.1", host: "127.0.0.1:8181"
